core/rfb.js

```diff
@@ -1788,6 +1772,34 @@ RFB.genDES = function (password, challenge) {
 };
 
 RFB.encodingHandlers = {
+    getHandler: function (rfb, encoding) {
+        switch (encoding) {
+            case encodings.encodingRaw:
+                return RFB.encodingHandlers.RAW.bind(rfb);
```

Comment:
IIRC, calling bind isn't a particularly fast operation.  Since this could be called in a hotloop, we really should pre-bind.

Reply:
I seem to recall it's the calling of bound functions that is slow (*was* slow most likely these days), so I'm not sure that will change much. However my suggestion on the previous comment would perform that change as a side effect.